### ISSUE TITLE
Gradle Convention Plugins

### ISSUE THREAD

**Issue by jeffscottbrown** [micronaut-projects/micronaut-starter#1363]
Should newly created projects which use Gradle be configured with Gradle convention plugin(s) by default?

**Comment by jeffscottbrown** [comment]
@melix I moved this question to an issue here at @timyates request.  Do you think that newly created projects which use Gradle should be configured with Gradle convvention plugin(s) by default?

**Comment by melix** [comment]
Yes, they probably should, but I wouldn't invest a lot of time on this before we finish some cleanup in the Micronaut Gradle plugins to make conventions easier to write. That's on my mind for some time.

**Comment by jeffscottbrown** [comment]
@melix Thank you for your feedback.  That is helpful.

Feel free to dispatch this ticket as appropriate, but I have what I need.  Thank you!
